### DISCUSSION TITLE
BLTouch does not stop when homing Z-axis 2.0.1

### DISCUSSION THREAD

**Discussion by 5P4P** [Jyers/Marlin#1601] [Q&A]
Hi,

Does anyone have the same problem? The probe does not register and just keeps on moving into the bed.
I was reading somewhere that people switch black and white wires on BLTouch.
I might revert back to 1.3.5b.

**Comment by 5P4P** [comment]
It stopped a few times now. I didnt change anything...

**Reply by 5P4P** [reply]
Well, it did not fail since yesterday anymore. I guess i will just keep using it and mentor what its doing.

**Reply by vhristenco** [reply]
I had this issue a while ago, on multiple firmware versions. Turned out to be an intermitent mechanical issue with probe's pin.
Anyway, since I replaced the pin the issue didn't appear anymore.

**Reply by vhristenco** [reply]
@tititopher68-dev that was a while ago... on 1.2.x I guess. Anyway, after trying everything I could imagine, I swapped the bltouch pin with the spare one and the issue disappeared.
